config.yml

```diff
@@ -11,139 +11,90 @@ rpcs:
   methods:
     - method: author_submitExtrinsic
       params:
-        - name: extrinsic
-          ty: Bytes
+        - Any
```

Comment:
the type should still be `Bytes`, but serialize into `Any` (or `Unknown` will be a better name)

Reply:
> the type should still be `Bytes`, but serialize into `Any` (or `Unknown` will be a better name)

Done